Vorstellung/README.md

```diff
@@ -0,0 +1,164 @@
+# Ein einfacher Pandoc/Markdown-Spickzettel
+
+pandoc ist ein Toolpaket zur Konvertierung zwischen Textformaten. Als
+Schriftsteller (und nicht als Techniker) finde ich es nützlich, um
+etwas, das in einfachem Text geschrieben wurde, in eine RTF-Datei zu
+konvertieren, die ich in einem Textverarbeitungsprogramm - bis hin zum
+gefürchteten Word - problemlos mit einigen Formatierungen lesen kann.
+Was ich hier beschreibe, ist nur ein winziger Bruchteil dessen, was man
+damit machen kann.
+
+Es ist so einfach wie:
+
+`$ pandoc -s datei.txt -o datei.rtf`
+
+(Ersetzen Sie \"rtf\" durch html usw., um etwas anderes zu erhalten).
+
+Es kann eine Datei konvertieren, die nur aus Tippen besteht, aber wenn
+Sie ein wenig einfache Formatierung verwenden, ziemlich genau eine
+Version der Markdown-Syntax, können Sie ein schönes RTF mit einigen
+nützlichen Zusatzfunktionen (zum Beispiel Kursivschrift) erhalten. Für
+eine Überschrift unterstreichen Sie mit Gleichheitszeichen oder
+Bindestrichen in der nächsten Zeile;
+
+    Überschrift hier
+    ------------
+
+oder
+
+    Überschrift hier
+    ============
+
+wenn sie verarbeitet werden, ergibt das eine Überschrift. (Für richtige
+H1 H2 usw. siehe unten).
+
+Die Absatzbildung wird durch Leerzeilen bestimmt, so dass dieser Text in
+der RTF-Datei einen Absatz ergibt und\...
+
+Dieser Text\
+enthält auch\
+einen Absatz\
+in der RTF-Datei.
+
+Wenn ich einen Return erzwingen will, beende ich eine Zeile mit einem
+Backslash.
+
+Dies funktioniert nicht innerhalb von Backtics (siehe unten), die die
+wortwörtliche Umgebung darstellen. Wenn ich also zum Beispiel einen
+Haufen Zeug wortwörtlich, ohne Zwischenabstände, haben möchte, beende
+ich jede Zeile mit einem Backslash *außerhalb der* Backtics:
+
+`Zeile 1''`\
+`Zeile 2''`\
+`` `Zeile 3` ``
+
+wird geben
+
+`Zeile 1`\
+`Zeile 2`\
+`Zeile 3`
+
+## Schriftarten formatieren
+
+Das Wort ist das Zeichen, das zur Abgrenzung des geänderten Textes
+verwendet wird. Der Code wird ebenfalls wörtlich angegeben.
+
+*Unterstrich* `_Unterstrich_`
+
+*Dollar* (mathematisch, was sich nicht in RTF übersetzen lässt)
```

Comment:
Etwas unverständlich würde LaTeX-Formeln in einen seperaten Teil tun und ein Beispiel für eine Formel geben eg. `$E = mc^{2}$`. Außerdem vielleicht noch zwischen inline, also `$E = mc^{2}$` und display math, also `$$E = mc^{2}$$` unterscheiden.